sdk/src/androidTest/java/com/hcaptcha/sdk/HCaptchaDialogFragmentTest.java

```diff
@@ -109,7 +115,7 @@ public void webViewReturnsError() throws Exception {
         final HCaptchaStateListener listener = new HCaptchaStateTestAdapter() {
             @Override
             void onFailure(HCaptchaException exception) {
-                assertEquals(HCaptchaError.SESSION_TIMEOUT, exception.getHCaptchaError());
+                assertEquals(HCaptchaError.CHALLENGE_ERROR, exception.getHCaptchaError());
```

Comment:
Why did the error type change from `SESSION_TIMEOUT` to `CHALLENGE_ERROR`?

Reply:
To keep test less confusing, because it's not test actually session timeout, but just some error